src/codeu/chat/server/Controller.java

```diff
@@ -189,6 +189,30 @@ public User newUser(Uuid id, String name, Time creationTime) {
     return user;
   }
 
+
+  @Override
+  public void addMember(Uuid user, Uuid conversation) {
+    ifIdFree(user) {
+      throw new IOException("Must add an existing user");
+    }
+    final User _user = model.userById().first(user);
```

Comment:
Why did you name the variable `_user`? Is that python thing. This isn't a good practice. At most, people use them to show that a variable is private, but it's still not a good choice. camelCase is typically always the way to go! 😄